cron/k8s/worker.yaml

```diff
@@ -44,13 +44,6 @@ spec:
                 secretKeyRef:
                   name: github
                   key: installation_id
-          resources:
-            requests:
-              memory: 5Gi
-              ephemeral-storage: 100Gi
-            limits:
-              memory: 12Gi
-              ephemeral-storage: 500Gi
```

Comment:
Does autoscaling work with `requests` deleted, but `limits` still there? If we remove `limits` do we risk consuming crazy resources if we have some resource leak?